Homework_3/homework_3.js

```diff
@@ -107,11 +107,17 @@ console.log('Ответ к уравнению 2: ' + x1 +' и ' + x2)
 
 // Task3**. Напишите программу, которая принимает целое положительное число n (одно любое число от 1 до 9), и выводит сумму равную 
 // n + nn + nnn, где n не перемножаются, а конкатенируются
-let n = 2
+
+let n = 1
 let aa = n
```

Comment:
By "simplify" I mean that you can assign `1' to `aa' and use it.

Reply:
@VladimirLQA but aa=n because n is checked in 'if' section